File: 백준/Silver/29615. 알파빌과 베타빌/알파빌과 베타빌.py
import sys
input = sys.stdin.readline

n, m = map(int, input().split())
num = list(map(int, input().split()))
wait = list(map(int, input().split()))

fixed_num = 0
for i in range(m):
    for j in range(m):
        if num[i] == wait[j]:
            fixed_num += 1

print(m - fixed_num)

File: 백준/Silver/29615. 알파빌과 베타빌/알파빌과 베타빌.js
const input = require("fs")
  .readFileSync(process.platform === "linux" ? "/dev/stdin" : "./example.txt")
  .toString()
  .trim()
  .split("\n")
  .map((v) => v.split(" ").map(Number));

const [[n, m], wait, frd] = input;

let chg = 0;
for (let i = 0; i < m; i++) {
  for (const v of frd) {
    if (wait[i] === v) {
      chg += 1;
    }
  }
}
console.log(m - chg);
